sdb/commands/zfs/btree.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class Btree(sdb.Walker):
+    """ walk  btree """
+
+    names = ["zfs_btree"]
+    input_type = "zfs_btree_t *"
+
+    def _val(self, start, size, idx):
```

Comment:
nit: regardless of what you decide to do with Matt's comment below could you add types to this method?

Reply:
Yup will do. Another thing I wanted to ask was whether I need to transform more of the Drgn objects I'm using into `int`s (like `elem_size`)? It seems to work as is, but I'm not sure about the correctness or best practices.